virt/virt-4-9-release-notes.adoc

```diff
@@ -23,6 +23,8 @@ include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
 {VirtProductName} guests can use the following operating systems:
 
 * Red Hat Enterprise Linux 6, 7, and 8.
+* Red Hat Enterprise Linux 9 Alpha (TP) and Red Hat Enterprise Linux 9 Beta (TP).
```

Comment:
```suggestion
* Red Hat Enterprise Linux 9 Alpha (TP).
```
In CNV 4.9.0 we provide a template for Rhel9 Alpha (which is the OS that was tested).